1-js/03-code-quality/04-ninja-code/article.md

```diff
@@ -223,6 +223,6 @@ Imagine, another developer wants only to check the email, and not output any mes
 
 All "pieces of advice" above are from the real code... Sometimes, written by experienced developers. Maybe even more experienced than you are ;)
 
-- Follow some of them -- and your code will become full of surprises.
-- Follow many of them -- and your code will become truly yours, no one would want to change it.
-- Follow all -- and your code will become a valuable lesson for young developers looking for enlightment.
+- Follow some of them, and your code will become full of surprises.
```

Comment:
@iliakan This change is correct punctuation, but as far as whether the sentence look better with dashes: that is a stylistic choice for you to make. But considering punctuation only, the "Follow..." clauses are **dependent clauses** of the rest of the sentence. In English when a dependent clause precedes an **independent clause**, we usually place a comma between them, although the comma is optional. It is considered incorrect to separate a dependent clause and an independent clause with a dash.